### PR TITLE
[SNG-DASH-2279] Change theme of code snippets in Sockets Registry to dark one

### PR DESCRIPTION
**Pull request by powerczuk**
screen shot 2016-12-30 at 16 22 09

### CODE REVIEW

File: src/apps/SocketsRegistry/DetailsView/SocketCodeExample.js [hunk 1]
```diff
@@ -66,7 +66,9 @@ const SocketCodeExamples = ({ socketName, currentLanguage, endpointName, methodT
       padding: '8px 24px',
       maxHeight: '400px',
       overflow: 'scroll',
-      backgroundColor: '#383838',
+      backgroundColor: '#272822',
+      borderRadius: '0.3em',
```
Comment (Flixow) [inline src/apps/SocketsRegistry/DetailsView/SocketCodeExample.js]: Why not in px?

Comment (powerczuk) [inline src/apps/SocketsRegistry/DetailsView/SocketCodeExample.js]: because example request also has a border-radius property in em

Comment (Flixow) [inline src/apps/SocketsRegistry/DetailsView/SocketCodeExample.js]: What do you mean?

Comment (powerczuk) [inline src/apps/SocketsRegistry/DetailsView/SocketCodeExample.js]: Example request container has the same property.

**Pull request merged**